cms/envs/common.py

```diff
@@ -476,7 +476,11 @@
     'openedx.core.djangoapps.header_control.middleware.HeaderControlMiddleware',
     'django.middleware.cache.UpdateCacheMiddleware',
     'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
+
+    # Replaced by Edraak to allow cross origin CSRF
+    # 'django.middleware.csrf.CsrfViewMiddleware',
+    'openedx.core.djangoapps.cors_csrf.middleware.CrossDomainCsrfViewMiddleware',
```

Comment:
Please undo all the CSRF changes and move it into another pull request. I'll follow up later with a detailed comment.

Reply:
Done. Moved to https://github.com/Edraak/edraak-platform/pull/92